Block/SentryScript.php

```diff
@@ -47,10 +47,6 @@ public function canUseScriptTag($blockName)
             return true;
         }
 
-        if ($this->useSessionReplay()) {
-            return true;
-        }
```

Comment:
This wouldn't actually have any effect as `useScriptTag` being false would prevent replays from loading anyways